mantidproject/mantid
# Added expanduser to Load in simpleapi

Description of work.

Added expansion of user's home directory (denoted by `~`) to `FileProperty`, so now files with `~` in the path can be loaded with standard behaviour, whereas previously they were not found.

**To test:**

Place a .nxs file in your home directory (~), or some subdirectory. From the script interpreter in MantidPlot, run `Load(Filename="~/<path-to-file>", OutputWorkspace="ws")`. The file should be loaded as expected.

Fixes #19068 . 

**Release Notes** 
[here](https://github.com/mantidproject/mantid/pull/20007/files#diff-d43ebaec89d7bc36364c8bf4c9f97673)


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
